extension.js

```diff
@@ -196,6 +192,14 @@ function executeCommand(commandInput, componentPath) {
 	});
 }
 
+function resolveNext(componentRequire) {
```

Comment:
I am confused how you intend for this to be used. In the function, you are catching the error and returning a value. But every usage, is ignoring the return value, and trying to catch its error.

Reply:
Ah I still don't have it working locally anyways. I'll follow up once it's ready to go